cypress/integration/visual-testing/button-test.spec.js

```diff
@@ -0,0 +1,48 @@
+/* global cy,before,after */
+import { environment } from '../../util'
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Carbon',
+      testName: 'Button',
+      browser: environment
+    })
+    cy.visit('http://localhost:3000')
```

Comment:
Thanks for this! For each of the `visit`'s do you need to prefix everything with `http://localhost:3000` or can you just use `cy.visit('/')`.

That way it will work with Cypress's `--baseUrl` flag

Reply:
I could have used the URL mentioned the cypress.json but that's pointing to production. So I was unable to use that.

Reply:
Can you no have `test:e2e` use `--baseUrl=http://localhost:3000`?

Or even just run it directly as `yarn test:e2e --baseUrl=http://localhost:3000`

I really appreciate you doing this work!

Reply:
I can run with these commands on my local but how will Travis pick this?
Do you want me to add another npm script for visual tests?